browser/src/control/jsdialog/Util.Dropdown.js

```diff
@@ -154,6 +154,9 @@ JSDialog.OpenDropdown = function (id, popupParent, entries, innerCallback, popup
 		};
 	};
 	L.Map.THIS.fire('jsdialog', {data: json, callback: generateCallback(entries)});
+
+	L.Map.THIS.fire('closepopups'); // close popups if a dropdown menu is opened
+	L.Map.THIS._docLayer._resetReferencesMarks();
```

Comment:
this should be done earlier (before jsdialog message which will create potential popup)

Reply:
I just put that before the `L.Map.THIS.fire('jsdialog', ....` line. I hope it is ok.